net/conserver/patches/002-addrsmatch-freeaddrinfo.patch

```diff
@@ -0,0 +1,37 @@
+diff -ru conserver-8.2.6.orig/conserver/consent.c conserver-8.2.6/conserver/consent.c
+--- conserver-8.2.6.orig/conserver/consent.c	2020-10-20 16:33:19.000000000 +1100
++++ conserver-8.2.6/conserver/consent.c	2024-02-15 20:10:07.318357509 +1100
```

Comment:
Is this patch included in upstream? If not, will you upstream it?
- There is no commit description, who is the author, etc.

Reply:
I am the author of the patch.  The tests complained about a "dirty patch" which so I removed the commit info thinking that was it, but it wasn't so I put it back.  Quoting from the first comment: "This was submitted upstream (https://github.com/bstansell/conserver/pull/98), not yet accepted."

Reply:
Then it should be done differently. Use it from this: https://github.com/bstansell/conserver/pull/98.patch and then refresh it.

Reply:
I think I've done that.  It's at least past the "dirty patch" check and seems to be building happily.

Reply:
Refreshing it (ie "make package/conserver/refresh V=s") drops the patch metadata from the start of the patch?

Reply:
I'm confused: are the patches suppose to have the patch metadata before the actual patch or not?   And if so, in what format?  Some patches in the repo do, but AFAICT (although I'm not 100% sure) all of the times I've tried with that it has failed.

Reply:
The patch has now been accepted upstream as https://github.com/bstansell/conserver/commit/47c232b88127354c6f79f766d8162583445cc6cf

The test failures seem unrelated to to this change, since it's nowhere near this change, and most other recent PRs are failing the same tests in the same ways.